foqus_lib/conftest.py

```diff
@@ -149,6 +149,7 @@ def pytest_terminal_summary(terminalreporter, config):
 
     tr.write_line(f"pytest temporary directory:\n\t{basetemp}")
     tr.write_line(f"subdirectories:")
-    for path in Path(basetemp).rglob("*"):
-        if path.is_dir():
-            tr.write_line(f"\t{path}")
+    if basetemp is not None:
```

Comment:
I added this line to prevent a long stream of log warnings printing after running tests locally, but now I'm not sure if the CI runs are excluding this since `codecov` flags it as uncovered.